iOS/Core/Fireproofing.swift

```diff
@@ -57,7 +57,6 @@ public class UserDefaultsFireproofing: Fireproofing {
     private var allowedDomainsIncludingDuckDuckGo: [String] {
         allowedDomains + [
             URL.ddg.host ?? "",
-            SubscriptionCookieManager.cookieDomain
```

Comment:
@miasma13 I was under impression that we still set something for subscription, but my knowledge is out of date - so just pointing this place to you to validate. :)

Reply:
@bwaresiak I was in touch with @brindy when he was planing out this work and I reconfirm that cookie route for sharing subscription route is not used (was briefly introduced but due to issues was rolled back and never introduced). The initial cookie based approach was replaced with more robust one using JS messaging based one.